index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="styles.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Favourite Meal</title>
+</head>
+<body>
+    <h1>Healthy Ramen</h1>
+    <br>
+
+    <p>Ramen is a Japnaese noodle dish. I love making this healthy ramen. It is easy to prepare and also super delicous. You should try it too. &#128516;</p>
+    <br>
+
+    <img src="Ramen1.jpeg" alt="Vegetable Ramen with runny half-boiled egg on top">
+    <br>
+
+    <h2>Ingredients</h2>
+    <br>
+
+    <h3>For Noodles</h3>
+    <ul>
+        <li>Brown rice ramen noodles (I do 4 cups of water for 2 noodle cakes)</li>
+        <li>2 cups of frozen or fresh veggies</li>
+        <li>Prawns</li>
+        <li>Optional: fresh parsley to garnish</li>
+    </ul>
+    <br>
+
+    <h3>For Seasoning</h3>
+    <ul>
+        <li>2 tsp veggie stock </li>
+        <li>1 tsp garlic powder</li>
+        <li>1 tsp onion powder</li>
+        <li>1/2 tsp chinese 5 spice powder</li>
```

Comment:
**Suggestion Only**

We can use HTML special characters if we want https://www.compart.com/en/unicode/U+00BD